apps/api/src/controllers/v1/scrape.ts

```
@@ -0,0 +1,149 @@
+        success: false,
+        error: `(Internal server error) - ${e && e.message ? e.message : e}`,
+      });
+    }
+  }
+
+  await getScrapeQueue().remove(jobId);
+
+  const endTime = new Date().getTime();
+  const timeTakenInSeconds = (endTime - startTime) / 1000;
+  const numTokens =
+    doc && doc.extract
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)